src/bin/tikv-server.rs

```diff
@@ -184,15 +183,11 @@ fn main() {
     info!("Start listening on {}...", addr);
     let listener = bind(&addr).unwrap();
 
-    let dsn_name = matches.opt_str("S").unwrap_or_else(|| MEM_DSN.to_owned());
+    let dsn_name = matches.opt_str("S").unwrap();
```

Comment:
Helper message should be updated too. And why not set the default dsn to rocksdb?